### PR TITLE
Poster Part 2 Redo: Putting Posters in Shipside 

### PR DESCRIPTION
**Pull request by novaepee**
## About The Pull Request

Big PR, lets go.

Add posters in PoS, Theseus, and Minerva. For Sulaco, I will wait for Derrick to do his thing.

Relocate TerraGov poster so that it is a sign and not have to cycle through 70+ posters.

Add more posters for flavor. One is a Wheel of Time reference. Nerd out.

Add directional posters to make facing a thing for posters.

## Why It's Good For The Game

Flavor and atmosphere! You don't think that marines and sailors are not going to have posters and pin ups in their barracks?

## Changelog
:cl:
add: Add posters in PoS, Theseus, and Minerva. For Sulaco, I will wait for Derrick to do his thing.
imageadd: Add more posters
imagedel: Delete redundant poster
code: directional poster!
/:cl:

### CODE REVIEW

File: code/game/objects/effects/decals/posters/bs12.dm [hunk 1]
```diff
@@ -233,6 +233,11 @@
 	name = "Security pinup"
 	desc = "This is a pin-up poster. A dark skinned white haired girl poses in the sunlight wearing a tank top with her stomach exposed. The text on the poster states \"M, Succubus of Security.\" and a lipstick mark stains the top right corner, as if kissed by the model herself."
 
+/datum/poster/bay_48
+	icon_state="bsposter48"
+	name = "L’Escrimeuse (The Fencing Girl)"
+	desc = "The picture of a nattily attired fencing girl, seated in the corner of a fencing room, with one knee gracefully crossed over the other, on her shapely breast the red mark which indicates to her opponent the heart, his vital mark is one of the most popular pictures of the late 19th century and early 20th century. It has gone into thousands of Western homes and clubs."
```
Comment (TiviPlus) [inline code/game/objects/effects/decals/posters/bs12.dm:239]: stop woith the horny

File: code/game/objects/structures/signs.dm [hunk 2]
```diff
@@ -557,3 +557,8 @@
 	name = "\improper TGMC poster"
 	desc = "An old recruitment poster for the TGMC. Looking at it floods you with a mixture of pride and sincere regret."
 	icon_state = "prop3"
+
+/obj/structure/sign/prop4
+	name = "TerraGov: United for Humanity"
+	desc = "A poster depicting TerraGov's logo and motto, reminding marines of who's looking out for humankind against the Xenomorph menace. Semper Fi."
```
Comment (TiviPlus) [inline code/game/objects/structures/signs.dm:563]: what semper fi whoo did you get this from

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/game/objects/effects/decals/posters/bs12.dm (hunk 1 above)
Comment (novaepee) [reply code/game/objects/effects/decals/posters/bs12.dm:239]: so sad

File: code/game/objects/structures/signs.dm (hunk 2 above)
Comment (novaepee) [reply code/game/objects/structures/signs.dm:563]: "According to marines, the term semper fi is a Latin phrase that is short for “semper fidelis.” This phrase means “always faithful.” Semper fi is the motto of every member of the U.S. Marine Corps. The United States Marine Corps uses the phrase to reflect a collective and eternal commitments to succeeding in battles, the [progress of the nation of the United States, and a loyalty to the Marine Corps at large. This phrase develops a bond between Marines and shows that being in the Marine Corps is not just a job, but a warriorhood. This phrase united individual marines and military units with their fellow Marines also performing military service, or who have served in the past."

Sprite is from TG Station.

Comment (TiviPlus) [reply code/game/objects/structures/signs.dm:563]: I can read wikipedia too but this is tgmc not the us marines

Comment (novaepee) [reply code/game/objects/structures/signs.dm:563]: SAD

**Review by TiviPlus — APPROVED** [review]

**Comment by TiviPlus** [comment]
@Kuro020

**Pull request merged**